tools/esdlcmd/esdlcmd_shell.cpp

```diff
@@ -180,7 +180,7 @@ bool EsdlCMDShell::parseCommandLineOptions(ArgvIterator &iter)
 void EsdlCMDShell::usage()
 {
     fprintf(stdout,"\nUsage:\n"
-        "    esdl [--version] <command> [<args>]\n\n"
```

Comment:
I could change the xsd and wsdl to use --ver_ (which nicely corresponds to the url parameter name to specify service version), and restore --version solely to printout build info. Do you guys like that better?